Source/WTF/wtf/text/StringCommon.h

```diff
@@ -344,13 +344,36 @@ ALWAYS_INLINE bool equal(const LChar* a, const UChar* b, unsigned length)
         }
         return true;
     }
-    // Otherwise, we just do a naive loop.
-#endif
+    if (length >= 4) {
```

Comment:
Since the cases < 8 don't use SIMD intrinsics, couldn't these also be enabled for x64 without further changes?

Reply:
The code is relying on that we have `length >= 8` loop, which x64 does not have.